module-2/backend/week_three.md

```diff
@@ -10,14 +10,31 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+  * rails -new
 2. What do Models generally inherit from in rails?
+  * ApplicationRecord::Base or ActiveRecord::Base
 3. What do Controllers generally inherit from in a rails project?
+  * ApplicationController
 4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+  * resources :horses, only:[:show]
 5. What rake task is useful when looking at routes, and what information does it give you?
+  * rake -T
```

Comment:
That's certainly helpful, but when looking at routes `rake routes` is usually my go to.